cmd/fluxctl/list_images_cmd.go

```diff
@@ -111,7 +114,7 @@ func (opts *imageListOpts) RunE(cmd *cobra.Command, args []string) error {
 			for _, available := range container.Available {
 				running := "|  "
 				_, _, tag := available.ID.Components()
-				if currentTag == tag {
+				if currentTag == tag && available.Digest == container.Current.Digest {
```

Comment:
IIRC, the value of the `Current` field is obtained by looking through the available images for the one that has the same ID (i.e., the same tag) -- so comparing the digests will trivially succeed when the tags are equal.

We don't have a definitive digest for the current image; there is no such thing, unless it's specifically named in the manifest.

Reply:
You are right, too bad, but understandable.